configs/anyrtc.json

```diff
@@ -1,10 +1,72 @@
 {
   "index_name": "anyrtc",
   "start_urls": [
-    "https://docs.anyrtc.io/rtc-android/docs/"
+    {
+      "url": "https://docs.anyrtc.io/",
+      "tags": ["all"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-web/",
+      "tags": ["rtc-web"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-android/",
+      "tags": ["rtc-android"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-ios/",
+      "tags": ["rtc-ios"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-window/",
+      "tags": ["rtc-window"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-web/",
+      "tags": ["rtm-web"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-android/",
+      "tags": ["rtm-android"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-ios/",
+      "tags": ["rtm-ios"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-window/",
+      "tags": ["rtm-window"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    }
   ],
   "sitemap_urls": [
-    "https://docs.anyrtc.io/rtc-android/sitemap.xml"
+    "https://docs.anyrtc.io/sitemap.xml"
```

Comment:
Please make sure this page is available. There is a 404 on this URL. Use https://docs.anyrtc.io/rtc-android/sitemap.xml instead since this page is available

Reply:
This point is not resolved. @chudongvip Can you clarify why you marked this as resolved?